rootfs/api/tests/test_pods.py

```diff
@@ -454,48 +502,55 @@ def test_command_good(self):
         app_id = response.data['id']
         app = App.objects.get(id=app_id)
         user = User.objects.get(username='autotest')
+
         # Heroku Buildpack app
-        build = Build.objects.create(owner=user,
-                                     app=app,
-                                     image="qwerty",
-                                     procfile={'web': 'node server.js',
-                                               'worker': 'node worker.js'},
-                                     sha='african-swallow',
-                                     dockerfile='')
+        build = Build.objects.create(
+            owner=user,
+            app=app,
+            image="qwerty",
+            procfile={
+                'web': 'node server.js',
+                'worker': 'node worker.js'
+            },
+            sha='african-swallow',
+            dockerfile=''
+        )
+
         # create an initial release
-        release = Release.objects.create(version=2,
-                                         owner=user,
-                                         app=app,
-                                         config=app.config_set.latest(),
-                                         build=build)
-        # create a container
-        c = Container.objects.create(owner=user,
-                                     app=app,
-                                     release=release,
-                                     type='web',
-                                     num=1)
+        Release.objects.create(
```

Comment:
there are a few instances below also, but I won't comment on them for brevity's sake